public/static/js/spawned_hosts.js

```diff
@@ -275,6 +272,25 @@ mciModule.controller('SpawnedHostsCtrl', ['$scope','$window', '$timeout', 'mciSp
       );
     };
 
+    $scope.updateTags = function() {
+        var tags_to_add = [];
+        for (key in $scope.curHostData.tags_to_add) {
+            var new_tag = key + "=" + $scope.curHostData.tags_to_add[key];
```

Comment:
Instead of concatenating them together as a string, can you send this down as a list of key:value pairs (so an object)? That way the data is more structured, and also = can be used in a tag value

Reply:
You're in London but I'll still respond... I could do that, but the validation server side expects it to be in this format, and also checks to make sure "=" isn't used in a tag/value, so I'd prefer to just format it here.